go.mod

```diff
@@ -3,8 +3,37 @@ module go.starlark.net
 go 1.13
 
 require (
+	github.com/NebulousLabs/go-upnp v0.0.0-20181203152547-b32978b8ccbf // indirect
```

Comment:
Lot of new dependencies. Run `go mod tidy`?

Reply:
Done.

Reply:
Thanks!